src/streamlink/plugins/bongacams.py

```diff
@@ -138,13 +129,18 @@ def _get_streams(self):
                                        self._get_stream_uid(stream_source_info['userData']['username'])),
             # Multiple args with same name not supported.
             # Details: https://github.com/streamlink/streamlink/issues/321
-            "conn" : "S:{username} --conn=S:{access_key} --conn=B:0 --conn=S:{data_key}".format(
+            "conn": "S:{username} --conn=S:{access_key} --conn=B:0 --conn=S:{data_key}".format(
                 username=stream_source_info['userData']['username'],
                 access_key=stream_source_info['localData']['NC_AccessKey'],
                 data_key=stream_source_info['localData']['dataKey']
             )
         }
 
+        subprocess_errorlog_path = self.session.options.options.get('subprocess-errorlog-path')
```

Comment:
I think better to add a separate option. `--subprocess-errorlog-debug` or something like that.
Because debug output very BIG: "Extremely verbose, including hex dumps of all packet data."

And in the case with `subprocess-errorlog-path`/`subprocess-errorlog` would be great to set `self.params["verbose"] = True`